sdk/purview/Azure.Analytics.Purview.Workflows/CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-10-26)
 
 ### Features Added
 
+- Add `/workflows/{id}/validate` API.
+- Add admin query options to workflow / workflow run APIs.
+
 ### Breaking Changes
 
-### Bugs Fixed
+- Split `PurviewWorkflowServiceClient` into multiple clients by model.
```

Comment:
Beta versions are expected to be unstable and have API changes to unreleased features.  We do not consider them breaking and avoid tracking them as such so that we don't add noise for customers looking for any GA-related breaking changes.    

Let's  move this to "Other changes"

Reply:
Thanks. Moved this.